examples/example_rdf_import_export.py

```diff
@@ -0,0 +1,80 @@
+"""An example explaining the RDF import and export functionality."""
+
+# Please install the city ontology: $pico install city
+
+# If you did not install the city ontology
+# (pico install city),
+# you have to execute these commands first:
+# from osp.core import Parser
+# p = Parser()
+# p.parse("city")
+
+import uuid
+import re
+import os
+from rdflib import URIRef
+from osp.wrappers.sqlite import SqliteSession
+from osp.core.namespaces import city
+from osp.core.utils import serialize_rdf_graph, branch, import_rdf_file, \
+    pretty_print
+
+uuid_re = re.compile(r".*(http://www\.osp-core\.com/cuds#"
+                     r"([a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}"
+                     r"-[a-z0-9]{4}-[a-z0-9]{12})).*")
+
+# Create CUDS structure
+c = branch(branch(city.City(name="Freiburg"),
+                  city.City(name="Pablo"),
+                  city.City(name="Yoav"),
+                  rel=city.hasInhabitant),
+           city.Neighborhood(name="Stühlinger"),
+           city.Neighborhood(name="Herdern"))
+
+# Export from Core Session
+serialize_rdf_graph("test.rdf", format="ttl")
+
+# Check output
+with open("test.rdf") as f:
+    print("Exported from Core Session")
+    for line in f:
+        print("\t", line.strip())
+
+# Export from a Wrapper session
+with SqliteSession(path="test.db") as session:
+    w = city.CityWrapper(session=session)
+    w.add(c)
+    serialize_rdf_graph("test.rdf", format="ttl", session=session)
+
+    # Check output
+    with open("test.rdf") as f:
+        print("Exported from SqliteSession")
+        for line in f:
+            print("\t", line.strip())
+
+    # Usually, RDF data does not contain UUIDs as in the current output.
+    # Replace UUIDs in the RDF file
+
+    print("Replace UUID in turtle file")
+    with open("test.rdf") as f1:
+        with open("test2.rdf", "w") as f2:
+            for line in f1:
+                match = uuid_re.match(line)
+                if match:
+                    uid = uuid.UUID(match[2])
+                    line = line.replace(match[1], "http://city.com/"
+                                        + session._registry.get(uid).name)
+                print("\t", line, end="")
+                print(line, end="", file=f2)
```

Comment:
I do not like this on an example, which should be as simple as possible.  I agree that usually in an RDF file you do not have UUIDs or ideally you would like to have a human-readable IRI that is self explanatory. But I would not expect users to really do this, and somehow this may mislead to the user to think that this must be done and/or that this is their task. This piece of code is not focused on showing how RDF import/export works, and actually it may be good that the user sees that OSP-core works with UUIDs.

If we go ahead in the future and allow the cuds to be identified by IRIs instead of URIs, then this would be solved. Another option could be to export CUDS as blank nodes, as this is somehow what they really are from a RDF point of view, individuals without an IRI.

Reply:
I agree, that this is not very nice. I thought either I create the RDF file to import by hand or I do this. I choose this, because then the user knows what to expect, because I only replace the some IRIs. If I just import any RDF file, we would have to explain that file first. 

Maybe it is enough to have a comment, that explains, that this is just for demonstration purposes? Something that the user would never have to do?
Otherwise I can also extract the RDF file after the replacement and put a comment there on how I created this file.

Reply:
Interestingly, it seems like BNodes can contain UUIDs:

```
>>> x = uuid.uuid4()
>>> g = rdflib.Graph()
>>> g.add((rdflib.BNode(x), rdflib.RDF.type, rdflib.OWL.Thing))
>>> g.serialize("test.ttl", format="ttl")
>>> g.serialize("test.json", format="json-ld")
>>> g.serialize("test.xml", format="xml")
```

Unfortunately the ttl export does not contain the UUID anymore, which is a no-go IMO:

```ttl
[] a <http://www.w3.org/2002/07/owl#Thing> .
```

The other serializations work, though:

```json
[
  {
    "@id": "_:68bf8f51-a8c3-4ad5-892f-83ae8f93d9b9",
    "@type": [
      "http://www.w3.org/2002/07/owl#Thing"
    ]
  }
]
```

```xml
<?xml version="1.0" encoding="UTF-8"?>
<rdf:RDF
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
>
  <rdf:Description rdf:nodeID="68bf8f51-a8c3-4ad5-892f-83ae8f93d9b9">
    <rdf:type rdf:resource="http://www.w3.org/2002/07/owl#Thing"/>
  </rdf:Description>
</rdf:RDF>
```